docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,105 @@
+How to enable esm-infra-legacy on Trusty
+*****************************************
+
+The ``esm-infra-legacy`` service can be enabled to grant additional two years of 
+extend security coverage. This service is only available on Trusty, since the release has reached
+its end of life support for esm-infra.
+
+
+Make sure ``ua`` is up-to-date
+===============================
+
+All systems come with ``ua`` pre-installed through the
+``ubuntu-advantage-tools`` package. To make sure that you're running the latest
+version of ``ua``, run the following commands:
+
+.. code-block:: bash
+
+    sudo apt update && sudo apt install ubuntu-advantage-tools
+
+
+Check the status of the services
+================================
+
+After you have attached to a Pro subscription and
+updated the ``ubuntu-advantage-tools`` package, you can check which services
+are enabled by running the following command:
+
+.. code-block:: bash
+
+    sudo ua status
+
+This will show you which services are enabled or disabled on your machine
+(output truncated for brevity):
+
+.. code-block:: text
+
+    SERVICE          ENTITLED  STATUS    DESCRIPTION
+    esm-infra        yes       enabled   Expanded Security Maintenance for Infrastructure
+    esm-infra-legacy yes       disabled  Expanded Security Maintenance for Infrastructure on Legacy Instances
+    livepatch        yes       enabled   Canonical Livepatch service
+
+If you are entitled to the service, you can enable it by running:
+
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+What to do if esm-infra-legacy is not entitled
+===============================================
+
+If status is showing that you are not entitled to the service, you
+will need to buy access to the service. You can contact sales/support for help on that regard.
```

Comment:
I would agree with that ordering - we wouldn't want users to get halfway through the article before realising they can't actually use it because they don't have the right subscription - the subscription is a pre-requisite so it should be mentioned at the very top. I also agree with minimising the "here to buy" locations and adding the blog post :+1: